### ISSUE TITLE
Accessibility: Drawer: aria-label attribute cannot be used on a span with no valid role attribute.

### ISSUE THREAD

**Issue by robertmausser** [adaptlearning/adapt_framework#3381]
### Subject of the issue/enhancement/features
.drawer-heading span uses aria-label attribute without valid role attribute

### Your environment
* version 5.8
* Chrome latest
* Windows 11

### Steps to reproduce
"Element Location:
.drawer-heading
<span class=""drawer-heading"" aria-label=""Top of side drawer""></span>"

### Expected behaviour
role="heading" should be applied

### Actual behaviour
No role
